### ISSUE TITLE
crnn can recognize line of charactors by my understading

### ISSUE THREAD

**Issue by mengzhibin** [qjadud1994/CRNN-Keras#8]
crnn can recognize line of charactors by my understading, but images of yours appears to be 2 lines.
is my understand right?

**Comment by qjadud1994** [comment]
I know that only one line of CRNN input is used for general text recognition.
However, because I applied CRNN only to license plate recognition, I think that it is possible to do 2 lines.

**Comment by qjadud1994** [comment]
Originally, images with two lines of text do not work well, but I can tell you two things. 
Since crnn is a structure that creates a height of 1 and a feature vector of width and channel, increasing the width expects to be able to distinguish the letters by the space between the upper and lower texts of the license plate. (It 's just my idea. )
And the other one is to increase the number of fake images a lot.

But it is unlikely to be a fundamental solution. It may be necessary to separate the top and bottom of the license plate by preprocessing

**Comment by jsn5** [comment]
@qjadud1994 Thanks for the reply.

**Comment by pawan9** [comment]
@jsn5 were you able train it for 2 line plates and get good results? 
If not how did you go about solving the 2 line plate ocr problem ?
 I am also trying to train this model on synthetic 2- line plate images (around 50k) but it doesn't learn anything, the loss hangs around 320.

**Comment by jsn5** [comment]
@pawan9 No, I was not able to get good results on 2 line plates, I ended up using a seperate model to split the lines and feed the individual lines to the OCR model.

**Comment by pawan9** [comment]
@jsn5 what kind of model did you use to seperate lines, was it a text detection one (something [like this](https://github.com/eragonruan/text-detection-ctpn)) or was it something else ?

**Comment by jsn5** [comment]
@pawan9 I used SSD mobilenet.

**Comment by pawan9** [comment]
@jsn5 Did you detect individual characters or words ? which approach do you think is good ?
